src/Sarif.Driver.UnitTests/ExceptionRaisingRule.cs

```diff
@@ -105,12 +105,19 @@ public void Analyze(TestAnalysisContext context)
 
             if (_exceptionCondition == ExceptionCondition.ParsingTarget)
             {
-                Errors.LogTargetParseError(context, null, "Could not parse target.");
+                Errors.LogTargetParseError(
+                    context,
+                    new Region
+                    {
+                        StartLine = 42,
+                        StartColumn = 54
+                    },
+                    "Could not parse target.");
```

Comment:
I added a real region here because the notification for target parsing errors -- which is the only notification in Errors.cs that actually is associated with a region in the source code -- actually does format the region into the notification message, and it null-refs if there isn't one.